apps/common-app/src/examples/RuntimeTests/ReJest/types.ts

```diff
@@ -53,15 +53,17 @@ export type TestCase = {
   | { decorator: Exclude<TestDecorator, TestDecorator.WARN | TestDecorator.FAILING> | null }
 );
 
+export type MaybeAsyncVoidFunction = () => void | Promise<void>;
```

Comment:
This type is already declared in this file in line 39: 
```
export type BuildFunction = () => void | Promise<void>;
```
Feel free to rename it if you think it doesn't match the new usages. If it is possible try finding something shorter, maybe `MaybeAsyncVoid` since it is easy to guess it must be a function.